src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -220,6 +236,33 @@ private void getEvents(HttpServletRequest request, HttpServletResponse response,
     return;
   }
 
+  private boolean joinEvent(HttpServletRequest request, HttpServletResponse response, UserService userService, String eventId)
+      throws IOException {
+    String currentUserId = null;
+    try {
+      currentUserId = userService.getCurrentUser().getUserId();
+    } catch (Exception e) {
+      System.err.println("Can't get current user id: " + e);
+      response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+      return false;
+    }
```

Comment:
Optional: I would move this to the doGet/doPost method and expect here a user id as param. Semantically this should not be part of joining an event.

Reply:
What do you mean? What do you want to move?

Reply:
The part of the code that I selected: 
```
   String currentUserId = null;
    try {
      currentUserId = userService.getCurrentUser().getUserId();
    } catch (Exception e) {
      System.err.println("Can't get current user id: " + e);
      response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
      return false;
    }
```